forge-gui/res/cardsfolder/upcoming/sword_of_wealth_and_power.txt

```diff
@@ -0,0 +1,12 @@
+Name:Sword of Wealth and Power
+ManaCost:3
+Types:Artifact Equipment
+K:Equip:2
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 2 | AddToughness$ 2 | AddSVar$ SwordOfWealthAndPowerCE | AddKeyword$ Protection:Instant:instants & Protection:Sorcery:sorceries | Description$ Equipped creature gets +2/+2 and has protection from instants and from sorceries.
```

Comment:
write it as single protection KW

Reply:
Done. I take it this is another cleanup item on the to-do list.

Reply:
@Hanmac 
we prefer to split it for the engine or have the text closer to the print?

Reply:
Have two Separate keywords would be easier (there is also a Rule about that)

Also haven't I updated the Plural style?
So shouldn't `Protection:Instant` be enough?

Reply:
FWIW, this is what I get with `AddKeyword$ Protection:Instant & Protection:Sorcery:sorceries`, assuming the plural style doesn't cover 'sorcery'.
![image](https://github.com/Card-Forge/forge/assets/45150760/666e38ce-04f6-4c3a-a7c1-e282d5910847)

Reply:
I think it should, try leaving that part away

Reply:
Turns out the plural style does cover 'sorcery'. Edited the relevant file accordingly.